src/fenced/fenced/disks.py

```diff
@@ -109,12 +108,17 @@ def reset_keys(self, newkey):
 
 class Disk(object):
 
-    def __init__(self, fence, name, pool=None):
+    def __init__(self, fence, name, is_nvme=False, pool=None):
         self.fence = fence
         self.name = name
         self.pool = pool
-        self.cam = cam.CamDevice(f'/dev/{name}')
         self.curkey = None
+        self.is_nvme = is_nvme
+
+        if self.is_nvme:
```

Comment:
Isnt the name enough for figuring out if its nvme? Why the is_nvme

Reply:
Just thought it looked a little better than doing `if name.startswith('nvd')` calls. I can change it to that easily enough.

Reply:
You're already sort of doing that when instantiating the Disk object, so if you dont change the API and keep it inside as an implementation detail will probably keep it saner

Reply:
Understood 👍